src/main/scala/org/ergoplatform/local/ErgoStatsCollector.scala

```diff
@@ -54,7 +55,9 @@ class ErgoStatsCollector(readersHolder: ActorRef,
     None
   )
 
-  override def receive: Receive = onConnectedPeers orElse getNodeInfo orElse onMempoolChanged orElse
+  var parameters: Parameters = LaunchParameters
```

Comment:
@kushti mutable state of an Actor better to be declared `private` to be consistent with other code.

Reply:
fixed